sktm/jenkins.py

```diff
@@ -18,25 +18,29 @@
 
 import jenkinsapi
 
-import sktm
+import sktm.misc
 
 
-class skt_jenkins(object):
-    """Jenkins interface"""
-    def __init__(self, url, username=None, password=None):
+class Project(object):
```

Comment:
I agree that it is confusing if you import the `Project` alone into an unrelated module. However, if you don't deal with Jenkins directly in the module you're importing it into like that, you shouldn't really be doing it. If you really have to, you can write `from sktm.jenkins import Project as JenkinsProject`.

I think adding `Jenkins` to the class name would be against the idea of Python's packaging system. I think it would be better to then just use `import sktm.jenkins as jenkins` and write `jenkins.Project`. Otherwise, as depth of hierarchy grows we can get to silly levels of duplication. We can already, kind of, write `SktmJenkinsProject`.

Reply:
If we can guarantee that it will be used in a way you mention here than it's fine I guess. I was just thinking about possibility of supporting different things in the future which might use similar names which would be confusing, since we already have several different things which are called 'project'.

Reply:
I think we can enforce that in our reviews, and I assume we would anyway. I think each module is responsible for taking care of ensuring that things make sense within its own namespace. One way is to simply always use global, fully-qualified identifiers, but if you'd like to have shorter names then something like this can be used:

    import sktm.jenkins as jenkins
    import sktm.patchwork as patchwork

    jp = jenkins.Project()
    pp = patchwork.Project()

Reply:
Readability wins, so this works :+1: (except the nondescriptive two-letter variable names, but we aren't talking about those here)